config/clusters/openscapes/common.values.yaml

```diff
@@ -34,207 +34,98 @@ basehub:
-              # NOTE: Node share choices are in active development, see comment
-              #       next to profileList: above.
-              display_name: Node share
-              choices:
-                mem_1:
-                  default: true
-                  display_name: ~1 GB, ~0.125 CPU
-                  kubespawner_override:
-                    mem_guarantee: 0.904G
-                    cpu_guarantee: 0.013
-                mem_2:
-                  display_name: ~2 GB, ~0.25 CPU
-                  kubespawner_override:
-                    mem_guarantee: 1.809G
-                    cpu_guarantee: 0.025
-                mem_4:
-                  display_name: ~4 GB, ~0.5 CPU
-                  kubespawner_override:
-                    mem_guarantee: 3.617G
-                    cpu_guarantee: 0.05
-                mem_8:
-                  display_name: ~8 GB, ~1.0 CPU
-                  kubespawner_override:
-                    mem_guarantee: 7.234G
-                    cpu_guarantee: 0.1
-                mem_16:
-                  display_name: ~16 GB, ~2.0 CPU
-                  kubespawner_override:
-                    mem_guarantee: 14.469G
-                    cpu_guarantee: 0.2
-                mem_32:
-                  display_name: ~32 GB, ~4.0 CPU
-                  kubespawner_override:
-                    mem_guarantee: 28.937G
-                    cpu_guarantee: 0.4
           kubespawner_override:
-            cpu_limit: null
-            mem_limit: null
-            node_selector:
-              node.kubernetes.io/instance-type: r5.xlarge
-        - display_name: "Medium: up to 16 CPU / 128 GB RAM"
-          description: *profile_list_description
-          slug: medium
-          profile_options:
-            image: *profile_options_image
+            image: openscapes/python:f577786
+          profile_options: &profile_options
             requests:
-              # NOTE: Node share choices are in active development, see comment
-              #       next to profileList: above.
-              display_name: Node share
+              display_name: Resource Allocation
               choices:
-                mem_1:
-                  display_name: ~1 GB, ~0.125 CPU
-                  kubespawner_override:
-                    mem_guarantee: 0.942G
-                    cpu_guarantee: 0.013
-                mem_2:
-                  display_name: ~2 GB, ~0.25 CPU
-                  kubespawner_override:
-                    mem_guarantee: 1.883G
-                    cpu_guarantee: 0.025
-                mem_4:
+                mem_1_9:
+                  display_name: 1.9 GB RAM, upto 3.75 CPUs
+                  kubespawner_override:
+                    mem_guarantee: 1992701952
+                    mem_limit: 1992701952
+                    cpu_guarantee: 0.234375
+                    cpu_limit: 3.75
+                    node_selector:
+                      node.kubernetes.io/instance-type: r5.xlarge
```

Comment:
This makes sense to have in a kubespawner_override on a higher level in a way, but at the same time, this is generated, so its acceptable to have it explicit here.

Reply:
Yes, the primary change here is to have the engineers and community champions make the choice of node size. I agree that during workshops it makes sense to pick a larger node size. Let's address that as part of https://github.com/2i2c-org/infrastructure/pull/3030 as well. 

This `node_selector` is in fact superfluous, and I've removed it.

Reply:
> I see now that the machine choice switches when a request requires 100% of a machine. So for example when ~32GB is required, should the user really run on a 32GB machine or should the user arrive on a 16 CPU 128GB machine instead perhaps?

For this particular resource allocation strategy (more about that in #3030), one assumption is that if you are requesting more resources, it's ok for your startup to take longer. Where should that cut-off be? That depends on the community! This should feed into selecting the kinds of nodes made available to users and be part of that guidance. Needs automation and documentation for sure.